### PR TITLE
BibTeX: add "\ " after "." in journal title

### PR DESCRIPTION
**Pull request by MJedr**
INSPIR-3246

### CODE REVIEW

File: backend/inspirehep/migrator/cli.py [hunk 1]
```diff
@@ -45,7 +45,10 @@ def touch_file(file):
     try:
         os.utime(file)
     except FileNotFoundError:
-        os.mknod(file)
+        try:
+            os.mknod(file)
+        except PermissionError:
+            open(file, 'w').close()
```
Comment (michamos) [inline backend/inspirehep/migrator/cli.py]: * this is sufficient as it always works:
```suggestion
        open(file, 'w').close()
```
* in principle, it should be in a separate commit, as it has nothing to do with BibTeX. If you don't know how to do, it's OK for this time.

**Comment by michamos** [comment]
Also backend/celerybeat-schedule.db was included in your commit by mistake.

**Comment by MJedr** [comment]
I forgot to rebase and change the commit message in the last commit, sorry
